### PR TITLE
Add a folder in the main repo for conferences attended with VDK

### PR DESCRIPTION
**Pull request by dvalkova**
VDK conferences: Add a folder in the main repo to store the "Welcome pages" of the conferences we attend

Having a folder inside VDK repo gives visibility to the community on which conferences did we present. It also gives a direct link between the main VDK repo and the repo we use for storing VDK examples/workshops used at the conferences themselves (https://github.com/versatile-data-kit-demo). Also, people at the conference could directly star the VDK repo instead of the workshops repo.

Added a folder "events" with subfolders for AMLD and OpenFest at which we already presented/will present. Inside each conf. subfolder, there's a README which is the "home/welcome" page shown on the conference and any other files that might be relevant.

Cosmetic change/folder structure

### CODE REVIEW

**Comment by tozka** [comment]
Let's name the folder with more genric name. I suggest "Events" because I can include the demo I did for the ML Community meetup as well - https://github.com/vdk-ml-community/vdk-demo

**Review by murphp15 — APPROVED** [review]

**Comment by dvalkova** [comment]
> Let's name the folder with more genric name. I suggest "Events" because I can include the demo I did for the ML Community meetup as well - https://github.com/vdk-ml-community/vdk-demo

Folder renamed from "conferences" to "events".

**Comment by doks5** [comment]
What is the idea of the `backup` folders? Also, the READMEs' for both conferences point to the same "local/backup" instructions not the ones in their respective backup folders, which makes me think we can maybe move the backup folder up in the main `events` folder and remove the backup folders from the separate conferences folders?

**Comment by dvalkova** [comment]
> What is the idea of the `backup` folders? Also, the READMEs' for both conferences point to the same "local/backup" instructions not the ones in their respective backup folders, which makes me think we can maybe move the backup folder up in the main `events` folder and remove the backup folders from the separate conferences folders?
> 
> Maybe also update the PR description to reflect the change in `conferences` folder name?

Agreed about the backup folders. A more appropriate place for them would be the https://github.com/versatile-data-kit-demo?tab=repositories account, so I will keep them only there, deleted them from this commit. 
I also updated the PR description.

File: events/amld-workshop/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    - [From Raw Data to a Cleaned Database: A Deep Dive into Versatile Data Kit](https://towardsdatascience.com/from-raw-data-to-a-cleaned-database-a-deep-dive-into-versatile-data-kit-ab5fd992a02e)
+## Workshop Solutions
+- [EV Range Prediction (Solved)](https://github.com/versatile-data-kit-amld/linear-regression-example-solved)
+- [Tracking Covid (Solved)](https://github.com/versatile-data-kit-amld/tracking-covid-cases-deaths-example-solved)
+- [Covid vs. No Scent Complaints (Solved)](https://github.com/versatile-data-kit-amld/correlation-example-solved)
+
+
+## Run workshop on local machine
+
+The workshops run in mybinder.org environment by default. But if you want to run it locally follow [instructions here](https://github.com/versatile-data-kit-amld/workshop/blob/main/backup/local_instructions.md)
```
Comment (doks5) [inline events/amld-workshop/README.md:72]: Where is this link supposed to point to?

File: events/openfest/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Relevant Articles
+    - [An Overview of Versatile Data Kit](https://towardsdatascience.com/an-overview-of-versatile-data-kit-a812cfb26de7)
+    - [From Raw Data to a Cleaned Database: A Deep Dive into Versatile Data Kit](https://towardsdatascience.com/from-raw-data-to-a-cleaned-database-a-deep-dive-into-versatile-data-kit-ab5fd992a02e)
+
+## Workshop Solutions
+- [Covid vs. No Scent Complaints (Solved)](https://github.com/versatile-data-kit-amld/correlation-example-solved)
+
+
+## Run workshop on local machine
+By default, the workshop runs in mybinder.org environment. If you want to run it locally, please follow [the instructions here](https://github.com/versatile-data-kit-amld/workshop/blob/main/backup/local_instructions.md).
```
Comment (doks5) [inline events/openfest/README.md:60]: Ditto.

File: events/amld-workshop/README.md (hunk 1 above)
Comment (dvalkova) [reply events/amld-workshop/README.md]: Just committed a change with updated link.

File: events/openfest/README.md (hunk 2 above)
Comment (dvalkova) [reply events/openfest/README.md]: Updated.

**Pull request merged**